x-pack/plugins/observability_solution/synthetics/server/feature.ts

```diff
@@ -63,7 +63,7 @@ const canManagePrivateLocationsPrivilege: SubFeaturePrivilegeGroupConfig = {
       includeIn: 'all',
       savedObject: {
         all: [privateLocationSavedObjectName, legacyPrivateLocationsSavedObjectName],
-        read: [],
+        read: [privateLocationSavedObjectName, legacyPrivateLocationsSavedObjectName],
```

Comment:
Do we need these since the parent feature `all` list them under `savedObject.read` as well?

Reply:
we don't, i think you reviewed outdated, i have already remove those